lib/src/view/settings/account_preferences_screen.dart

```diff
@@ -520,6 +537,67 @@ class _PieceNotationSettingsScreenState extends ConsumerState<PieceNotationSetti
   }
 }
 
+class ShowRatingsSettingsScreen extends ConsumerStatefulWidget {
+  const ShowRatingsSettingsScreen({super.key});
+
+  @override
+  ConsumerState<ShowRatingsSettingsScreen> createState() => _ShowRatingsSettingsScreenState();
+}
+
+class _ShowRatingsSettingsScreenState extends ConsumerState<ShowRatingsSettingsScreen> {
+  Future<void>? _pendingSetShowRatings;
+
+  @override
+  Widget build(BuildContext context) {
+    final accountPrefs = ref.watch(accountPreferencesProvider);
+    return accountPrefs.when(
+      data: (data) {
+        if (data == null) {
+          return Center(child: Text(context.l10n.mobileMustBeLoggedIn));
+        }
+
+        return FutureBuilder(
+          future: _pendingSetShowRatings,
+          builder: (context, snapshot) {
+            return CupertinoPageScaffold(
+              navigationBar: CupertinoNavigationBar(
+                trailing:
+                    snapshot.connectionState == ConnectionState.waiting
+                        ? const CircularProgressIndicator.adaptive()
+                        : null,
+              ),
+              child: SafeArea(
```

Comment:
This `SafeArea` is useless and prevents the header blur effect so it should be removed.

Reply:
Done :+1:  Maybe it's time to do some refactoring and add a common widget for these setting choice screens? I can have a look in the next few days

Reply:
Sure.